lib/testers/test-parser.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @author Toru Nagashima <https://github.com/mysticatea>
+ */
+"use strict";
+
+const espree = require("espree");
+const Traverser = require("../util/traverser");
+
+/**
+ * Remove `start`/`end` properties from the given node.
+ * @param {ASTNode} node The node to remove.
```

Comment:
We're no longer removing the `start`/`end` properties, but rather redefining them as getters that throw. Does this JSDoc need to be changed?